src/channel.ts

```diff
@@ -162,11 +172,24 @@ export function send<MESSAGE_TYPE extends keyof ShopwareMessageTypes>(
 export function handle<MESSAGE_TYPE extends keyof ShopwareMessageTypes>
```

Comment:
Remove the export because we are using only the `handleListener` now. It could be the case that you change the tests so that they use the the factory in the beginning